src/desktop/components/fair_week_marketing/PageScaffold.js

```diff
@@ -0,0 +1,192 @@
+import React from 'react'
+import styled, { ThemeProvider } from 'styled-components'
+
+import colors from 'reaction/Assets/Colors'
+import { Row, Col } from 'reaction/Components/Grid'
+import Text from 'reaction/Components/Text'
+import Title from 'reaction/Components/Title'
+
+const Container = styled.div`
+  margin: 0 auto;
+  max-width: 1192px;
+  padding-top: 25px;
+  @media (max-width: 48em) {
+    padding: 10px 20px 0;
+  }
+`
+
+const SectionTitle = Title.extend`
+  margin-top: 0;
+  line-height: 1;
+`
+
+const IntroductionText = Text.extend`
+  line-height: 31px;
+  @media (max-width: 24em) {
+    font-size: 20px;
+    line-height: 26px;
+  }
+`
+
+const FairLogo = styled.img`
+  width: 100%;
+  display: inline;
+  @media (min-width: 48em) {
+    max-width: 160px;
+  }
+`
+
+const ResponsiveRow = styled(Row)`
+  @media (max-width: 48em) {
+    margin-left: -8px;
+    margin-right: -8px;
+  }
+`
+
+const ReveredColumnOnMobile = styled.div`
+  @media (max-width: 48em) {
+    display: flex;
+    flex-direction: column-reverse;
+    img {
+      margin-left: auto;
+      margin-right: auto;
+    }
+  }
+`
+
+const theme = {
+  flexboxgrid: {
+    breakpoints: {
+      // em, not pixels
+      xs: 0,
+      sm: 24,
+      md: 48,
+      lg: 64,
+    },
+  },
+}
+
+export default ({
+  introduction,
+  fair_coverage,
+  event,
+  prepare_for_fairs,
+  displayStickyFooter,
+}) => (
+  <ThemeProvider theme={theme}>
+    <Container>
+      <ResponsiveRow style={{ paddingBottom: 50 }}>
```

Comment:
Maybe this is just preference, but I like to stick to `styled-components` and pass in additional properties as props instead of mixing the two styling methods. Not necessarily a blocker tho!